sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntity.cs

```diff
@@ -7,7 +7,10 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// Health care entity class.
+    /// A model which contains info about the detected health care entity.
+    /// A healthcare entity represented as a node in a directed graph
+    /// where the edges are a particular type of relationship between
+    /// the source and target nodes.
```

Comment:
I see JS has this, but I think this only makes sense in the context of the relationships. Maybe we should just remove it